.npmrc

```diff
@@ -0,0 +1,3 @@
+# will force display errors if there is a version mismatch
+#  with node or npm defined in the package.json
+engine-strict=true
```

Comment:
What is the effect if we don't have `engine-strict=true`? Can we display warnings without failing the build?

Reply:
It force displays the errors in the console, we need this in order to see them, the .npmrc file defines the settings on how NPM should behave when running commands.

Reply:
That's helpful, thanks. Do you know if it's possible to make it display warnings but not errors?

Reply:
I think I answered a different question than the one you asked, i'll look into this because i'm not sure

Reply:
My thought though is that adding the strict true is better because it's alerting a problem before it starts, if we make it a warning, it may end up breaking the build later and just kicks the problem down the road.  If this is a warning, i'm not sure I see the point of changing this at all...warnings get ignored (like the ones on the main page of this repo warning us about vulnerabilities that still have gone unchanged)

Reply:
I may not be looking at the benefits of having a warning correctly, but the way I'm seeing it, just making it a warning so it passes the build isn't the best justification, but I am totally open to being wrong about this